modules/tensor_mechanics/test/tests/lagrange/total/convergence/2D/dirichlet.i

```diff
@@ -0,0 +1,130 @@
+# Simple 2D plane strain test
+
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+  large_kinematics = true
+[]
+
+[Variables]
+      [./disp_x]
+      [../]
+      [./disp_y]
+      [../]
+[]
+
+[Mesh]
+  [./msh]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 4
+    ny = 4
+  []
+[]
+
+[Kernels]
+  [./sdx]
+      type = TotalLagrangianStressDivergence
+      variable = disp_x
+      component = 0
+  [../]
+  [./sdy]
+      type = TotalLagrangianStressDivergence
+      variable = disp_y
+      component = 1
+  [../]
+[]
+
+[Functions]
+  [./pullx]
+    type = ParsedFunction
+    value ='0.5 * t'
+  [../]
+  [./pully]
+    type = ParsedFunction
+    value ='-0.3 * t'
+  [../]
+[]
+
+[BCs]
+  [./leftx]
+    type = DirichletBC
+     preset = true
+    boundary = left
+    variable = disp_x
+    value = 0.0
+  [../]
+  [./lefty]
+    type = DirichletBC
+     preset = true
+    boundary = left
+    variable = disp_y
+    value = 0.0
+  [../]
+  [./pull_x]
+    type = FunctionDirichletBC
+    boundary = right
+    variable = disp_x
+    function = pullx
```

Comment:
In case you are not aware of this shortcut yet, you can directly say `function = '0.5 * t'` here and moose will create a function for you at run time.

Reply:
Cool, I didn't know that.